### PR TITLE
[ShapeOpt] surface normal shape update response

### PR DESCRIPTION
**Pull request by armingeiser**
This is a simple geometric response function that can be used to maximize/minimize the surface normal component of the shape update. 

It can be used in combination with the packaging response to wrap the design surface smoothly around an object. 

large_small_front

### CODE REVIEW

**Review by philbucher — DISMISSED** [review]
looks good to me :+1: 

Up to you if you want to wait for approval of others

**Review by philbucher — DISMISSED** [review]

File: applications/ShapeOptimizationApplication/python_scripts/response_functions/surface_normal_shape_change.py
Comment (MFusseder) [inline applications/ShapeOptimizationApplication/python_scripts/response_functions/surface_normal_shape_change.py]: response function singular

File: applications/ShapeOptimizationApplication/python_scripts/response_functions/surface_normal_shape_change.py
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/response_functions/surface_normal_shape_change.py:49]: May be this can be refactored? I see this is being used few times in other py scripts.

**Review by sunethwarna — DISMISSED** [review]
just a suggestion

**Comment by MFusseder** [comment]
I only found a typo. The overall implementation looks fine.

File: applications/ShapeOptimizationApplication/python_scripts/response_functions/surface_normal_shape_change.py
Comment (armingeiser) [reply applications/ShapeOptimizationApplication/python_scripts/response_functions/surface_normal_shape_change.py]:
```suggestion
    This simple geometric response function calculates the sum of all shape updates in
```

File: applications/ShapeOptimizationApplication/python_scripts/response_functions/surface_normal_shape_change.py
Comment (armingeiser) [reply applications/ShapeOptimizationApplication/python_scripts/response_functions/surface_normal_shape_change.py:49]: Thats true. I will create a new Issue/PR for this

**Review by philbucher — APPROVED** [review]

**Comment by loumalouomega** [comment]
@armingeiser can we delete the branch?

**Pull request merged**
